scripts/docker-compose/imports/openzaak-database/database/4-setup-zac-config-after.sql

```diff
@@ -10,4 +10,4 @@ INSERT INTO documenten_enkelvoudiginformatieobject (id, identificatie, bronorgan
 -- Set up the BAG service configuration. This requires that the corresponding variables have been passed on to this script.
 INSERT INTO zgw_consumers_service (label, api_type, api_root, client_id, secret, auth_type, header_key, header_value, oas, nlx, user_id, user_representation, oas_file, client_certificate_id, server_certificate_id, uuid) VALUES ('BAG', 'orc', :'BAG_API_CLIENT_MP_REST_URL', '', '', 'api_key', 'X-Api-Key', :'BAG_API_KEY', :'BAG_API_CLIENT_MP_REST_URL', '', '', '', '', NULL, NULL, 'd986d53f-68a9-45af-a3ff-194b1b8ec089');
 -- Set up the Objecten service configuration.
-INSERT INTO zgw_consumers_service (label, api_type, api_root, client_id, secret, auth_type, header_key, header_value, oas, nlx, user_id, user_representation, oas_file, client_certificate_id, server_certificate_id, uuid) VALUES ('Objects API', 'orc', 'http://objecten-api.local:8000/api/v2/', '', '', 'api_key', 'Authorization', 'Token cd63e158f3aca276ef284e3033d020a22899c728', 'http://objecten-api.local:8000/api/v2/schema/openapi.yaml', '', '', '', '', NULL, NULL, '9fe5f3fb-a1f5-4ea5-a2fe-c5b1294da0f4');
+INSERT INTO zgw_consumers_service (label, api_type, api_root, client_id, secret, auth_type, header_key, header_value, oas, nlx, user_id, user_representation, oas_file, client_certificate_id, server_certificate_id, uuid) VALUES ('Objects API', 'orc', 'http://host.docker.internal:8010/api/v2/', '', '', 'api_key', 'Authorization', 'Token cd63e158f3aca276ef284e3033d020a22899c728', 'http://host.docker.internal:8010/api/v2/schema/openapi.yaml', '', '', '', '', NULL, NULL, '9fe5f3fb-a1f5-4ea5-a2fe-c5b1294da0f4');
```

Comment:
Hmm. I'm surprised this even works because `host.docker.internal` definitely is not reachable from Docker Compose when running our integration tests in GitHub. That is the reason why we don't use `host.docker.internal` in any of there startup scripts. Really the use of `host.docker.internal` should be restricted where we really needed and not here I think. It is a workaround for running things locally really.

Reply:
Or maybe `host.docker.internal` is reachable by GitHub with @hsiliev 's Linux trick now..? I dunno.. But even then in these scripts we should not use it I think. I see no need for it. Just stick with the internal Docker container names.

Reply:
That's probably why the tests are failing now. Should we then add a developer instruction to map this via their host file? So `objecten-api.local` -> `localhost:8010`?

Reply:
Not sure. This is pretty standard Docker Compose port mapping stuff right? The same for all other mapped ports we have.

Reply:
The thing is, when we run ZAC from IntelliJ, it's not part of the docker compose setup, so it can't reach the containers by host name. I'll see if I can get it to work by changing my host file.

Reply:
ZAC is not but Objecten is still part of the Docker Compose setup in that situation.

Reply:
I'm not getting my point across, let's have a call about this later.